components/Prize/PrizeItems.jsx

```diff
@@ -0,0 +1,24 @@
+require('./PrizeItem.scss')
+
+import React from 'react'
+import PrizeItem from './PrizeItem'
+
+const PrizeItems = ({type,title,items}) => {
+  return (
+    <div className="PrizeSection">
```

Comment:
Can you please rename class name to match the component name i.e. `PrizeItems` ?